src/components/Navigation.spec.js

```diff
@@ -3,17 +3,14 @@ import { MemoryRouter } from 'react-router-dom';
 import Navigation from './Navigation';
 
 describe('Navigation', () => {
-  it('renders a navigation with one link', () => {
+  it('renders a navigation with two links', () => {
     render(
       <MemoryRouter>
         <Navigation />
       </MemoryRouter>
     );
 
-    const vocabularycardsLink = screen.getByRole('link', {
-      name: /Vocabulary Cards/i,
-    });
-
-    expect(vocabularycardsLink).toBeInTheDocument();
+    const navigation = screen.getByRole('navigation');
```

Comment:
I think you should test if the two links are also there and not only the navigation in general or keep the two links out if the it part 😊